assets/RVC/Cloud/EasyGUI.md

```diff
@@ -376,7 +376,7 @@ ii. <u>[TB](https://docs.aihub.wtf/rvc/resources/epochs--tensorboard/#tensorboar
 ‎   
 
 !!!warning While training, you might get disconnected if you:    
-- <u>[Stay AFK](https://rentry.org/colab_workarounds)</u> for a long time.     
+- <u>[Stay AFK](https://docs.google.com/document/d/1Pr-AZndodmWgsbOeuHQU4IrgbatFgYc1ChOq_ZAf_5s/edit?usp=sharing)</u> for a long time.     

```

Comment:
edit url??

Reply:
Yes, because the old rentry guide is outdated and all the images don't work anymore, and I can't change it, so I re made my old guide